### ISSUE TITLE
Use grafana sdk instead of a forked repo

### ISSUE THREAD

**Issue by mvalletta** [netsage-project/grafana-dashboard-manager#24]
Hi @safaci2000 
can you use https://github.com/grafana-tools/sdk instead of your forked repo https://github.com/netsage-project/sdk ?
I think it's better for security reasons

**Comment by safaci2000** [comment]
I don't mind doing so though they've been really slow at handling PRs.  I need to go back and see what all the changes are.  I need several patches to be accepted before we can switch over.  the netsage fork is ahead and fixed some behavior that prevented some features from working.  Like this [PR](https://github.com/grafana-tools/sdk/pull/146) been open since May

**Comment by mvalletta** [comment]
> Why do you think it's a security issue?
I can trust grafana/sdk but I can't of netsage-project/sdk because grafana-dashboard-manager has few downloads and I don't known you.
This require an extra audit on external library like grafana/sdk and this is a waste of time.

**Comment by safaci2000** [comment]
I think that's the point I was trying to make.  It's not an official SDK from the grafana community.  It's under grafana-tools which isn't owned by grafana it's a group of individuals that wrote a useful wrappers around the Grafana API.  It's not different than if I create an org called grafana-official and put it under that namespace.  It won't make it any more official than netsage-project or grafana-tools.  

To quote their Org:  "Tools and libraries for helping work with Grafana (http://grafana.org). This is a community effort with no official relations to Grafana project."

---

Basically If we can get more traction on the grafana-tools I'm happy to use it but since they seem to be pretty slow on getting changes in, I need to use the fork.  You can point it to grafana-tools but you'll get compiler errors due to patches still missing. 

Long term, might revisit what we're actually getting out of the SDK and if it's even worth pulling that in or if it would be easier to  re-implement the features we need in the grafana-tools itself.

**Comment by mvalletta** [comment]
I agree with all you wrote.
The only difference I saw between your fork and the master are the number of watch, star and fork. Larger is the community, larger is the trustworthiness , now I don't have other parameters to evaluate a project.
But I hope will be good news on the https://github.com/grafana-tools/sdk/issues/152.

>Long term, might revisit what we're actually getting out of the SDK and if it's even worth pulling that in or if it would be easier to re-implement the features we need in the grafana-tools itself.
I think this is another valid option.

**Comment by safaci2000** [comment]
I added one more PR for the dashboard import I think that's the big one that broke things.  I'll take another pass to make sure that's all and if we can come up with an easy way to flip back and forth as needed, i'm fine with staying closer to upstream if we can.

**Comment by safaci2000** [comment]
This has been addressed.  Please see #31
